cheqd/pyproject.toml

```diff
@@ -0,0 +1,92 @@
+[tool.poetry]
+name = "cheqd"
+version = "0.1.0"
+description = ""
+authors = []
+
+[tool.poetry.dependencies]
+python = "^3.12"
+cryptography = "<38.0.0"
+
+# Define ACA-Py as an optional/extra dependancy so it can be
+# explicitly installed with the plugin if desired.
+# acapy-agent = { version = "~1.1.0", optional = true }
+acapy-agent = {git = "https://github.com/cheqd/aries-cloudagent-python.git", branch = "develop"}
```

Comment:
I guess this can't be updated until the new acapy version gets released. This is fine for now. It will get updated during the next release process.